.travis.yml

```diff
@@ -1,15 +1,19 @@
 ---
-
+dist: xenial
 language: python
-python: "2.7.13"
+python: "3.7"
 sudo: required
-dist: trusty
 
 services:
   - docker
 install:
   - pip install pipenv
-  - pipenv install -r test-requirements.txt --python 2.7
+  - pipenv sync
+env:
+  matrix:
+    - MOLECULE_DISTRO: debian10
```

Comment:
Should Redis 3.x and Redis 4.x installation be tested?

Reply:
Good point ;-)